docs/core/examples/java/TransactionBasics.java

```diff
@@ -115,14 +117,20 @@ public static void main(String[] args) throws Exception {
       // endsnippet
     }
 
-    // snippet create-bob-multiasset-program
-    bobProgram = new ControlProgram.Builder()
-      .controlWithAccountByAlias("bob")
+    // snippet create-bob-multiasset-receiver
+    Receiver bobGoldReceiver = new Account.ReceiverBuilder()
```

Comment:
I thought about that too, but I think the batch syntax is a little too distracting/power-usery for this context. I actually wouldn't recommend using a batch call if you were creating only two receivers.